docs/UserGuide.md

```diff
@@ -94,7 +94,7 @@ _Refer to the [Features](#features) section for details of each command!_
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a popup window with a link to the User Guide.
```

Comment:
maybe add "clickable" in front of link just to make it very clear how it's been improved